src/stan/mcmc/hmc/static_uniform/adapt_dense_e_static_uniform.hpp

```diff
@@ -0,0 +1,53 @@
+#ifndef STAN_MCMC_HMC_STATIC_UNIFORM_ADAPT_DENSE_E_STATIC_UNIFORM_HPP
+#define STAN_MCMC_HMC_STATIC_UNIFORM_ADAPT_DENSE_E_STATIC_UNIFORM_HPP
+
+#include <stan/mcmc/stepsize_covar_adapter.hpp>
+#include <stan/mcmc/hmc/static_uniform/dense_e_static_uniform.hpp>
+
+namespace stan {
+  namespace mcmc {
+    // Hamiltonian Monte Carlo on a
```

Comment:
This should be a class comment rather than `//` for the doc.  No big deal given our doc is so poor for this stuff, but we ought to at least move in the right direction here.